evap/staff/views.py

```diff
@@ -1925,7 +1940,7 @@ def user_index(request):
         .order_by("last_name", "first_name", "email")
     )
 
-    return render(request, "staff_user_index.html", dict(users=users, filter_users=filter_users))
+    return render(request, "staff_user_list.html", dict(users=users, filter_users=filter_users))
```

Comment:
codecov says this was not covered by tests -- it seems like there was no test before for the user index page, so it would be nice if you could add a simple one that just loads the page and asserts that there is no server error.